Group_Constitution

```diff
@@ -0,0 +1,127 @@
+Constitution of “Mozilla Science Study Group- University of Toronto”
+
+1.	Name
+
+The official name of this recognized campus group is “     ”
+
+The acronym or abbreviation of this group is:      
+
+2.	Purpose and Objectives
+
+The purpose, objectives, mission and/or mandate of organization is outlined here:
+     
+
+3.	Membership
+
+Membership to the group is open to all the University of Toronto members (students, staff, faculty and alumni). 
+
+U of T members are permitted to run, nominate, and vote in elections and constitutional amendments. The group is open to non-U of T members. However, these members do not hold the aforementioned rights. Members must register with a designated executive by submitting their full name and a valid email address.
+
+The membership fee will be $      per year.
+
+Any member of the club may apply for a refund within one (1) month of becoming a member. No refunds are permitted 30 days after the election periods.
+
+For recognition by the University of Toronto Students’ Union (UTSU), the group must maintain a minimum of 30 members, a total of 51% of membership are UTSU members. The group must also maintain recognition from the Office of Student Life. These requirements are subject to change and should be checked with UTSU annually to ensure qualifications are met. 
```

Comment:
@QuLogic: You're absolutely right. We won't be using this constitution to gain recognition from UTSU. This will be used to apply to ULife for campus recognition by the Office of Student Life. (I know these organizations all sound like the same thing but somehow they are different). The GSU doesn't have the same formal application that the UTSU has, so when we get recognized by the ULife office, we can appeal to the GSU for funds/support.